java/ql/integration-tests/all-platforms/java/buildless-inherit-trust-store/test.ql

```diff
@@ -0,0 +1,8 @@
+import java
+import semmle.code.java.Diagnostics
+
+query predicate diagnostics(Diagnostic d) { any() }
+
+from Class c
+where c.getName() = "DepClass"
```

Comment:
Buildless question: is this guaranteed to only exist if the dependency was extracted successfully or would this exist merely because it is mentioned in `Test.java`? I assume it is the former, but wanted to check.

Reply:
A stub symbol would exist just because of the mention, but we would also get a diagnostic raised.

Reply:
So the correctness of the test hinges on the absence of the diagnostic? Possibly worth mentioning somewhere as well in a comment.